cluster-provision/images/vm-image-builder/test-containerdisk.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+set -ex
+
+PROJECT_DIR="$(
+  cd "$(dirname "$BASH_SOURCE/../")"
+  pwd
+)"
+
+SCRIPT_DIR="$(
+  cd "$(dirname "$BASH_SOURCE")"
+  pwd
+)"
+
+function cleanup() {
+    echo "[test-containerdisk] cleanup"
+    if [ -z $DEBUG ]; then
+      kubectl delete vmi $VMI_NAME --ignore-not-found=true || true
+      pushd $kubevirt_dir
+        make cluster-down
+      popd
+      rm -rf $kubevirt_dir
+      rm -rf "${IMAGE_NAME}_build"
+    fi
+}
+
+readonly KUBEVIRT_URL="https://github.com/kubevirt/kubevirt"
+
+trap 'cleanup' EXIT SIGINT
```

Comment:
I do not think you need SIGINT, EXIT should cover exit reasons. (please confirm)